docs/docs-content/vm-management/vm-management.md

```diff
@@ -10,112 +10,48 @@ sidebar_custom_props:
-
-  :::warning
+![A drawing of VMs deployed to Palette](/vm-mangement_vmo-diagram.webp)
 
-  In environments that use nested virtualization, where VMs operate inside of VMs due to lack of hardware to host VMs,
-  it is technically possible to operate VMs in Kubernetes by setting the KubeVirt resource `useEmulation` to true.
-  However, we do not recommend this approach.
-
-  :::
-
-## Get Started With VM Orchestrator
-
-To get started, review [Virtual Machine Orchestrator Pack](vm-packs-profiles/vm-packs-profiles.md) to learn about its
-components.
-
-Review [Create a VMO Profile](vm-packs-profiles/create-vmo-profile.md) and
-[Add Roles and Role Bindings](vm-packs-profiles/add-roles-and-role-bindings.md) to learn how to create the cluster
-profile and add roles and permissions that allow users to create and manage Virtual Machines (VMs).
-
-Palette VM Orchestrator provides various methods to quickly deploy VMs from out-of-the-box templates or from your
-organization's templates. To learn more about using and creating templates, review
-[Deploy VM From a Template](create-manage-vm/standard-vm-operations/deploy-vm-from-template.md) and
-[Create a VM Template](create-manage-vm/create-vm-template.md).
+## Use Cases
 
-While you can import template disks from external locations when deploying a VM, it can be inefficient. Instead, you can
-leverage the `DataVolume` resource to import template disks once and then clone them when deploying new VMs. Review the
-[Create Disk Templates](create-manage-vm/create-disk-templates.md) guide for more information.
+You will benefit from Palette VMO in the following cases:
 
-## Feature Gates
+- You are planning to shift from VMs to containers gradually and want to continue running both during the transition.
 
-Palette VM Orchestrator utilizes open-source KubeVirt as a component of the **Virtual Machine Orchestrator** pack to
-manage VMs and enables the following KubeVirt feature gates by default:
+- Your established infrastructure combines containers and VMs, and you want to manage them more effectively.
 
-- LiveMigration
-- Snapshot
-- HotplugVolumes
-- VMExport
-- ExpandDisks
-- HotplugNICs
-- VMLiveUpdateFeatures
+- You are integrating new VM-based applications into an existing containerized infrastructure.
 
-KubeVirt offers other feature gates you may find useful and which you can enable using
-[Kubernetes feature gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/). To enable
-more KubeVirt feature gates, you can modify the `kubevirt.kubevirtResource.additonalFeatureGates` parameter in the
-**Virtual Machine Orchestrator** manifest.
+- You are managing edge locations with VM-based workloads and would like to stop using a hypervisor.
 
-For more information on KubeVirt feature gates, refer to the
-[KubeVirt user guide](https://kubevirt.io/user-guide/cluster_admin/activating_feature_gates/).
+## Get Started
```

Comment:
Nice high-level overview. I like that the path is laid out for the reader of what to read 👏🏻